index.html

```diff
@@ -101,11 +148,48 @@
         }
       });
 
+      // The view for the lists container
+      // Should create a new todos container view
+      var ListsContainerView = Backbone.View.extend({
```

Comment:
Since this already contains a collection, you should look into Marionette's CollectionView's. It'll automatically handle adding/removing ItemViews that are tied to models inside the collection